source/AI.cpp

```diff
@@ -2379,8 +2384,8 @@ void AI::DoSurveillance(Ship &ship, Command &command, shared_ptr<Ship> &target)
 		bool cargoScan = ship.Attributes().Get("cargo scan power");
 		bool outfitScan = ship.Attributes().Get("outfit scan power");
 		// If the pointer to the target ship exists, it is targetable and in-system.
-		bool mustScanCargo = cargoScan && !Has(ship, target, ShipEvent::SCAN_CARGO);
-		bool mustScanOutfits = outfitScan && !Has(ship, target, ShipEvent::SCAN_OUTFITS);
+		bool mustScanCargo = cargoScan && !target->CargoScanCompletedBy(gov) && !Has(ship, target, ShipEvent::SCAN_CARGO);
+		bool mustScanOutfits = outfitScan && !target->OutfitScanCompletedBy(gov) && !Has(ship, target, ShipEvent::SCAN_OUTFITS);
```

Comment:
```suggestion
		bool mustScanCargo = cargoScan && !target->CargoScanCompletedBy(gov)
				&& !Has(ship, target, ShipEvent::SCAN_CARGO);
		bool mustScanOutfits = outfitScan && !target->OutfitScanCompletedBy(gov)
				&& !Has(ship, target, ShipEvent::SCAN_OUTFITS);
```
Fixing line wraps. Technically only the second one is necessary, but I think it is more readable with both. As usual, the exact position or the wrap is somewhat arbitrary.

Reply:
Yeah, in my code formatter it did the wrap and I reverted it.  Hard for me to determine if it’s more readable or not.

Reply:
That being said I don’t mind the suggestion.